proposals/b:zoe1/core-eval-support.js

```diff
@@ -0,0 +1,152 @@
+// @ts-check
+import {
+  Far,
+  makeMarshal,
+  makeTranslationTable,
+} from '../../upgrade-test-scripts/lib/unmarshal.js';
+import { Fail, NonNullish } from '../../upgrade-test-scripts/lib/assert.js';
+
+// TODO: factor out ambient authority from these
+// or at least allow caller to supply authority.
```

Comment:
not my words! :) 

future work https://github.com/Agoric/agoric-3-proposals/issues/21